classroom.rb

```diff
@@ -0,0 +1,52 @@
+require_relative './student'
+require_relative './person'
+require_relative './classroom'
+
+class Classroom
+  def initialize(label)
+    @label = label
+  end
+  attr_accessor :label, :students
```

Comment:
- [ ] **Optional:** Add rentals to the accessor to ensure it can be accessed/gotten.